lms/djangoapps/courseware/tabs.py

```diff
@@ -383,6 +383,12 @@ def get_course_tab_list(user, course):
     # further investigation since CourseTabList.iterate_displayable returns
     # Static Tabs that are not returned by the CourseTabPluginManager.
     course_tab_list.sort(key=lambda tab: tab.priority or float('inf'))
+
+    if course_tab_list:
+        from lms.djangoapps.course_goals.models import UserActivity
```

Comment:
import currently here because it was causing problems for cms tests which i think weren't getting the goals migration or something - but will look into if i can avoid this